config/deploy.rb

```diff
@@ -44,11 +44,12 @@
       end
     end
   end
-  
+
   desc "Restart zartan-related monitored processes"
   after :seed_config, :restart do
     on roles(:web) do
-      if test("ps cax | grep monit")
+      enabled_file = File.join(fetch(:deploy_to), 'shared', 'enabled')
```

Comment:
Could you document this in [Readme.md](Readme.md) and add a comment to that effect above this line?